libcontainer/README.md

```diff
@@ -155,7 +159,7 @@ config := &configs.Config{
 		Parent: "system",
 		Resources: &configs.Resources{
 			MemorySwappiness: nil,
-			Devices:          specconv.AllowedDevices,
+			Devices:          devices
```

Comment:
Needs a comma at the end, otherwise this code won't compile.

Reply:
I fixed it and  squashed the commits!